recipe/meta.yaml

```diff
@@ -27,14 +28,15 @@ requirements:
     - setuptools_scm
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - packaging
 
 test:
   commands:
     - python -c "import bcolz; bcolz.test()"
   requires:
     - numpy
     - setuptools
+    - packaging
```

Comment:
If packaging is in the run deps you shouldn't need to duplicate in the test deps (it won't hurt, but it's not necessary)
```suggestion
```